Revamp README

i-c-u-p opened the pull request:
Changed formatting of the README and added proper tables (used HTML so I could make headerless tables and use colspan on tables with headers)

i-c-u-p committed:
Revamp README

Changed formatting of the README and added proper tables (used HTML so I could make headerless tables and use colspan on tables with headers)

i-c-u-p committed:
Update README.md

replace <i> tags with *

i-c-u-p committed:
README table indentation fix

fixes the lack of indentation for the first three tables

i-c-u-p committed:
convert colspan tables to markdown

i-c-u-p committed:
bring back the rest of the doc (oops)

bruh

i-c-u-p committed:
table indent fix

first 3 tables had an extra indent

i-c-u-p committed:
Update README.md

Another indent fix; i need to use preview more

i-c-u-p committed:
Replace ☑ with 🤔️❓️ in README

Replaces the usage of "☑" with "🤔️❓️" for representing FAQ questions to match the look of [colemak.org](https://colemak.org/) closer.

DreymaR commented:
Oh. You treated the colemak_org README as a Markdown document that is read by the public. This is indeed the case with most repo readmes, but this particular one wasn't intended for that. What the public should read is the web page, and the repo exists to provide the web page. Therefore I've used check boxes to mark off which points I've finished adding to the page, and I've used html tags in the lines because they are meant for the web page. I have not intended this file as a proper Markdown formatted document.

Do you think the README should stand on its own as a human-readable document? I fear that'll create confusion and double effort. I didn't mean for this file to be scrutinized by the public, but I don't think there is a way of hiding it while keeping the web page visible?

i-c-u-p commented:
> Oh. You treated the colemak_org README as a Markdown document that is read by the public. This is indeed the case with most repo readmes, but this particular one wasn't intended for that. What the public should read is the web page, and the repo exists to provide the web page. Therefore I've used check boxes to mark off which points I've finished adding to the page, and I've used html tags in the lines because they are meant for the web page. I have not intended this file as a proper Markdown formatted document.
> 
> Do you think the README should stand on its own as a human-readable document? I fear that'll create confusion and double effort. I didn't mean for this file to be scrutinized by the public, but I don't think there is a way of hiding it while keeping the web page visible?

I think instead of the checkmarks the README could have something like the "TODO/WIP" and "DONE" sections in your repo for the BigBag website, where TODO contains a list of things to be added to the website, and stuff that has already been added goes in the "DONE" section. The stuff that's currently checkmarked should also probably be summarized first before being put into a DONE section since atm is mostly just a copy of what's on colemak.org. One part of the README that doesn't seem to be on the website yet is the tables, so those could go in the TODO.

i-c-u-p committed:
Revert changing the headers to use hashtags

i-c-u-p committed:
remove content already on site and add todo

Removed content that was already on the site, unless one of it's sub points wasn't on the site in which case the sub point is marked with a "❌️" emoji and the parent point is kept for context. Also added a TODO that starts with an explanation for how the emojis are used
